cypress.json

```diff
@@ -1,10 +1,10 @@
 {
   "baseUrl": "http://localhost:57021",
   "integrationFolder": "./src",
-  "testFiles": "!(tests)/**/*.test.@(ts|tsx)",
+  "testFiles": "@(apps|components)/**/*.test.@(ts|tsx)",
```

Comment:
This reraises my question from last week: 

> I wonder if we could place test files along with the code that they test like we do for Cypress tests?

> I guess there is a possibility for mixing up the two frameworks if all test files use the format `[scope].test.ts`. If we want to separate the two then one approach might be to use vite for `src/core` and Cypress for `src/apps` and `src/components`. That would also clearly communicate the responsibility of the different tools and folders. It might be too strict though.

With this change we seem to be halfway there anyway.